### ISSUE TITLE
Paper-checkbox stays coloured after deselection in object

### ISSUE THREAD

**Issue by bvhme** [PolymerElements/paper-checkbox#102]
I have an object from firebase-document with booleans inside it and am using two way databinding to update them. If the object changes the checkmark and its value change accordingly but the background-color of the checkbox is never changed.

**Comment by notwaldorf** [comment]
I think there's something incorrect in your setup. Here's a JSBin with a bunch of two-way-binded items, and the checkbox selects and deselects correctly: http://jsbin.com/lahoyi/edit?html,output.

Going to close this since I can't repro, but please re-open if you can provide an actual example where this is broken and that we can debug :(

**Comment by bvhme** [comment]
Try this one, put `a` into the input, then try `b`, then `c` then `d`

**Comment by notwaldorf** [comment]
It looks ok? 
![2016-02-03 15_42_42](https://cloud.githubusercontent.com/assets/1369170/12800830/c7ed7abe-ca8c-11e5-8d4f-ae8b8a75d148.gif)

**Comment by notwaldorf** [comment]
Interesting, but if i type 'bb' I see what you mean. Hmm.

**Comment by bvhme** [comment]
It's the only element that seems to do this, and I've run into it in multiple projects, all related to the firebase objects though being hotswapped out

**Comment by notwaldorf** [comment]
When I type `bb`, `_computeCheckboxClass: function(checked, invalid)` isn't called, which means that `checked` doesn't actually change. What does `object.checked3` actually return? Is it `undefined` by any chance?

**Comment by notwaldorf** [comment]
(as in: when I'm debugging, it seems that when I type `bb` in your demo, `checked` is set to undefined, which makes sense why the `_computeCheckboxClass` method isn't called)

**Comment by bvhme** [comment]
Yeah, it returns `null` I think, or is just not defined, which probably doesn't kick off the computed property...

**Comment by bvhme** [comment]
@notwaldorf Should this be reopened or is it not a bug?

**Comment by notwaldorf** [comment]
I don't think it's really a bug, since it looks like you're using the attribute incorrectly (setting `checked='undefined'` is not correct). 

But I do agree that maybe we should, for consistency, style in css using attributes vs. using a computed class, which would at least make the element consistent with the radio-button, for example.

I'll re-open and retitle
